pycloudlib/oci/utils.py

```diff
@@ -37,6 +37,7 @@ def get_subnet_id(
     network_client: "oci.core.VirtualNetworkClient",  # type: ignore
     compartment_id: str,
     availability_domain: str,
+    vcn_name: str = None,
```

Comment:
```suggestion
    vcn_name: Optional[str] = None,
```

And the appropriate import if required.

Reply:
Fixed, tysm :D